README.md

```diff
@@ -1,5 +1,8 @@
 # jetcd - A Java Client for etcd
-[![Build Status](https://img.shields.io/travis/coreos/jetcd/master.svg?style=flat-square)](https://travis-ci.org/coreos/jetcd) [![License](https://img.shields.io/badge/Licence-Apache%202.0-blue.svg?style=flat-square)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Maven Central](https://img.shields.io/maven-central/v/coreos/jetcd.svg?style=flat-square)](http://http://search.maven.org/#search%7Cga%7C1%7Ccom.coreos) [![GitHub release](https://img.shields.io/github/release/coreos/jetcd.svg?style=flat-square)](https://github.com/coreos/jetcd/releases)
+[![Build Status](https://img.shields.io/travis/coreos/jetcd/master.svg?style=flat-square)](https://travis-ci.org/coreos/jetcd)
+[![License](https://img.shields.io/badge/Licence-Apache%202.0-blue.svg?style=flat-square)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![Maven Central](https://img.shields.io/maven-central/v/coreos/jetcd.svg?style=flat-square)](https://search.maven.org/#search%7Cga%7C1%7Ccoreos)
```

Comment:
This should be something like: 

    https://img.shields.io/maven-central/v/com/coreos/jetcd-core.svg?style=flat-square

(my mistake)